ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -67,12 +67,12 @@
 
 | Interface | Description | Type | Vlan ID | Dot1q VLAN Tag |
 | --------- | ----------- | -----| ------- | -------------- |
-{%         for ethernet_interface in encapsulation_dot1q_interfaces | arista.avd.natural_sort %}
-{%             set description = ethernet_interfaces[ethernet_interface].description | arista.avd.default('-') %}
-{%             set type = ethernet_interfaces[ethernet_interface].type %}
-{%             set vlan_id = ethernet_interfaces[ethernet_interface].vlan_id | arista.avd.default('-') %}
-{%             set encapsulation_dot1q_vlan = ethernet_interfaces[ethernet_interface].encapsulation_dot1q_vlan | arista.avd.default('-') %}
-| {{ ethernet_interface }} | {{ description }} | {{ type }} | {{ vlan_id }} | {{ encapsulation_dot1q_vlan }} |
+{%         for ethernet_interface in encapsulation_dot1q_interfaces | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```

Comment:
the conversion and sorting is not needed here, since you build the `encapsulation_dot1q_interfaces` list above in line 53-62. With these changes the `encapsulation_dot1q_interfaces` goes from being a list of keys to a list of dicts. Same for flexencap_interfaces below.